pkg/controller/performanceprofile/components/profile/profile.go

```diff
@@ -3,7 +3,7 @@ package profile
 import (
 	"fmt"
 
-	"github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1"
+	v1 "github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1"
```

Comment:
It seems the preferred convention is now performancev1, just saying

Reply:
in general if it the API that defined under the same repo as a controller, I think you do not need the `performance` prefix, at least we had such convention under the KubeVirt
If it the API from the different repo you should add the prefix

Reply:
my take: the biggest concern is ambiguity. We do renamed imports (can't remember the go-specific name atm) when we need to do something like

```
import (
    "github.com/foo/pkg/great/api/v1"
    "github.com/bar/pkg/another/great/api/v1"
)
```
and we need to disambiguate the import. In this case there is no ambiguity, so from my perspective code is fine.
We can drop the initial "v1" - we are aliasing the old package with its own name, but I don't mind.

Reply:
ok